cumulusci/core/github.py

```diff
@@ -444,6 +453,12 @@ def format_github3_exception(exc: Union[ResponseError, TransportError]) -> str:
         sso_error_msg = check_github_sso_auth(exc)
         user_warning = scope_error_msg + sso_error_msg
 
+    if isinstance(exc, ConnectionError):
+        if "self signed certificate" in str(exc.exception):
+            user_warning = SELF_SIGNED_WARNING
+        else:
+            return None  # type: ignore
```

Comment:
Ah, I was too... `Literal`. Should have said empty string
```suggestion
            return ''
```